lock/gremlin_consul_comb_lock.go

```diff
@@ -54,10 +56,17 @@ func (lock ConsulCombinationLock) Lock() error {
 }
 
 func (lock ConsulCombinationLock) Unlock() error {
+	errResponse := ""
 	err := lock.ConsulLock.Unlock()
 	if err != nil {
-		return err
+		errResponse = fmt.Sprintf("Consul unlock error: %v", err)
 	}
 	err = lock.LocalLock.Unlock()
-	return err
+	if err != nil {
+		errResponse = fmt.Sprintf("%s. Local unlock error: %v", errResponse, err)
+	}
+	if err != nil {
+		return fmt.Errorf(errResponse)
+	}
```

Comment:
huh, why not combine the above two err!=nil checks into one

Reply:
Because that's a typo and the second one should say errResponse != nil